package/com.unity.formats.usd/Tests/Editor/UsdAssetReloadTests.cs

```diff
@@ -109,6 +117,38 @@ public void UsdAsset_Reload_FileDidNotChange_ValuesDoNotChange()
             Assert.AreEqual(new Vector4(1, 1, 1, 1), translate);
         }
 
+        [Test]
+        [Ignore("USDU-199")]
+        public void UsdInstancerAsset_Reload_NoDuplicateObjects()
+        {
+            var instancerChildNamePrefix = "Root";
+
+            var scene = OpenUSDGUIDAssetScene(k_USDInstancerGUID, out _);
+            var instancerObject = ImportHelpers.ImportSceneAsGameObject(scene);
+
+            var originalChildrenCount = instancerObject.transform.childCount;
```

Comment:
I think it would be good to store the expected value from the USDA as a const in the test (I believe the file defines 100 point instances), else if something broke and the value was wrong (zero, for example) this test would miss it.

Reply:
sorry about the messy PR, i've rebased it with the new updated origin/dev so it should be cleaner